controllers/grafana/grafana_controller.go

```diff
@@ -37,7 +37,7 @@ var log = logf.Log.WithName(ControllerName)
 // +kubebuilder:rbac:groups=integreatly.org,resources=grafanas;grafanas/finalizers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=integreatly.org,resources=grafanas/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=extensions;apps,resources=deployments;deployments/finalizers,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create
+// +kubebuilder:rbac:groups="",resources=pods;events,verbs=get;list;watch;create
```

Comment:
Why do we need to control pods? If we create them shouldn't we delete them as well?
To my knowledge we are always using the deployment API and that will handle anything happening to the pods.
Do we do anything specific towards a pod as a controller?

Reply:
I added this after seeing an error message in the operator log for not being able to get it's own pod. I'm assuming the operator sdk needs that to create the metrics endpoints.

Reply:
Ahh okay, do we really need create then? 
Shouldn't `get;list;watch` be okay?

Reply:
:+1:  create is not needed here